packages/nodes-base/nodes/Google/Sheet/v2/actions/sheet/update.operation.ts

```diff
@@ -283,13 +283,13 @@ export async function execute(
 	const keyIndex = columnNames.indexOf(columnsToMatchOn[0]);
 
 	//not used when updating row
-	const columnValues = await sheet.getColumnValues(
+	const columnValues = await sheet.getColumnValues({
 		range,
 		keyIndex,
-		firstDataRow,
+		dataStartRowIndex: firstDataRow,
```

Comment:
same as before could you add names for the rest of the parameters to make the code more consistent and readable?